### ISSUE TITLE
is there a way that I can make more than 5 buffs show up at once?

### ISSUE THREAD

**Issue by wayser** [obble/modui#154]
is there a way to make all of the buffs show up rather than just five at once? i've tried just about everything to do this, had friends that don't know that much about LUA try to help me, reddit , etc. nothing seems to work :(. is there any way you could point me in the right direction? this addon is flawless and a staple of my UI, this seems to be the only issue i've ever really had with it!

**Comment by Steffen-Hadberg-Nielsen** [comment]
https://github.com/obble/modui/issues/147

**Comment by wayser** [comment]
i see that you posted about this too but did you get it fixed cruzix?

**Comment by Steffen-Hadberg-Nielsen** [comment]
Nope, @obble has been gone from this project ever since the Elysium drama happened sadly... :/ 

I really crave this update and the "toggle features in game" update that he was working on, though.

**Comment by Steffen-Hadberg-Nielsen** [comment]
@wayser tbh I feel like our best bet is for someone else to fix this, because I feel like he just completely abandoned this project since he haven't said anything anywhere.

**Comment by wayser** [comment]
@Cruzix i've got a fix for an older version that someone on reddit did for me but i think it's kinda just a temp one. i've tried to find people to do it D:

**Comment by Steffen-Hadberg-Nielsen** [comment]
Please share if you find a fix! 👍

**Comment by 62574353** [comment]
@wayser you have this fix available?

**Comment by Finrodi** [comment]
Would love a fix for  that as well
